documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our release notes, and this directory contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
+
+* ``feature``: New user-facing feature.
+* ``bugfix``: A bug fix.
+* ``doc``: Documentation improvement.
+* ``removal``: Deprecation or removal of public API.
```

Comment:
API -> behavior

(API is a common term for libraries, but cocotb has more than that, e.g. the Makefiles)

Reply:
```suggestion
* ``removal``: Deprecation or removal of public API or behavior.
```